### DISCUSSION TITLE
ResidentialModule_01_Colony clothes cabinet doesnt have tags (it's very important)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13944] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

ResidentialModule_01_Colony clothes cabinet doesnt have tags so it doesn't spawn items.

Maybe the most niche ticket ever made but I'm using it for a custom event it's very important believe me also could you add one or two more around the module 🙏 

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/6ece63fa-cb81-4f1e-b52b-5456dc82a681)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/3ab2175d-c03d-4ffc-bb64-2baceca0a32f)

### Reproduction steps

Visit a city in campaign
Loot cabinet
It's empty

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

**Comment by itchyOwl** [comment]
Added the tag in FakeFishGames/Barotrauma-development@3a72f01.

**Reply by ubertpendragon** [reply]
Can you add one more in one of the rooms 🙏 ? it's not much but it will help one of my events believe me. Smaller outposts have like 3 of those
